be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -466,6 +460,26 @@ void PipelineDriver::_close_operators(RuntimeState* runtime_state) {
     }
 }
 
+void PipelineDriver::_set_operator_execute_mode(RuntimeState* state, MemTracker* tracker, OperatorPtr& op,
+                                                const ChunkPtr& chunk) {
+    // TODO: FIXME
+    // a simple spill stragety
+    if (state->enable_spill() && !op->is_max_performance_level() &&
+        op->revocable_mem_bytes() > state->spill_operator_min_bytes()) {
+        auto request_reserved = 0;
+        if (chunk == nullptr) {
```

Comment:
this if is not necessary, just use `request_reserved = op->estimated_memory_reserved(chunk);` is ok

Reply:
estimated_memory_reserved() means estimated memory reserved for set_finishing

Reply:
see the detail implements for Operator::estimated_memory_reserved